src/pages/debrief/cat-be/debrief.cat-be.selector.ts

```diff
@@ -84,16 +84,17 @@ export const getVehicleCheckDangerousFaults =
     if (!vehicleChecks) {
       return result;
     }
+    const faults = vehicleChecks.showMeQuestions.filter(fault => fault.outcome === CompetencyOutcome.D);
```

Comment:
Can we retrofit this to CAT B ?

Reply:
Similar logic already exists in catb.  it will only push a result if the show me question outcome is D.